modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/serviceextensions/preupdatepassword/PreUpdatePasswordActionBaseTestCase.java

```diff
@@ -401,14 +423,28 @@ protected void updateFlowStatus(String flowType, boolean enable) throws Exceptio
         flowManagementClient.updateFlowConfig(flowConfigDTO);
     }
 
-    protected void addRegistrationFlow(FlowManagementClient client) throws Exception {
+    protected void addFlow(FlowManagementClient client, String flowResourcePath) throws Exception {
```

Comment:
Since there are wrapper methods for this, either make this private or remove the wrapper methods